### PR TITLE
tutorial for adding radius to existing application

### PR DESCRIPTION
**Pull request by willtsai**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.radapp.dev/community/contributing/docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Contributing a new tutorial for adding radius to existing application.

### Summary
📝🚀🎓

This pull request adds a new tutorial document `tutorial-add-radius/index.md` to the docs repository. The tutorial shows how to use annotations to add Radius to a Kubernetes application. The tutorial is intended for users who want to learn how to use Radius features incrementally.

> _`Radius` tutorial_
> _Adding to Kubernetes app_
> _Spring of annotations_

### Walkthrough
* Add a new tutorial document `docs/content/tutorials/tutorial-add-radius/index.md` that teaches users how to add Radius to an existing Kubernetes application using annotations ([link](https://github.com/radius-project/docs/pull/870/files?diff=unified&w=0#diff-433b2d9477841bd80ba09f366a1da3c98ba29d0141642d946f7122a64f442b8eL1-R302))

## Issue reference

dependent upon this PR being merged: https://github.com/radius-project/samples/pull/626

### CODE REVIEW

File: docs/content/tutorials/tutorial-add-radius/index.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+
+## Prerequisites
+
+- [Kubernetes cluster]({{< ref "guides/operations/kubernetes/overview#supported-kubernetes-clusters" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref howto-rad-cli >}})
+
+## Overview
+
+As a part of this tutorial you will deploy an existing and containerized Guestbook application originally authored by the Kubernetes community for use in their own tutorials, then add Radius to the deployed application. To incrementally add Radius to an existing application, you will leverage the built-in Kubernetes integration functionality by adding an annotation to the application's Kubernetes deployment manifest. This approach is particularly useful for adding Radius capabilities to applications that are already deployed.
```
Comment (jonvsm) [inline docs/content/tutorials/tutorial-add-radius/index.md]:
```suggestion
As a part of this tutorial you will deploy an existing containerized Guestbook application originally authored by the Kubernetes community for use in their own tutorials. You will then add Radius to the deployed application. To incrementally add Radius to an existing application, you will leverage the built-in Kubernetes integration functionality by adding an annotation to the application's Kubernetes deployment manifest. This approach is particularly useful for adding Radius capabilities to applications that are already deployed.
```

File: docs/content/tutorials/tutorial-add-radius/index.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+   Open a browser and navigate to [`http://localhost:8080`](http://localhost:8080). You should see the Guestbook application running:
+
+   <img src="./guestbook-ui.png" alt="Guestbook application" width=500>
+
+   Terminate the port forwarding process by pressing `CTRL+C` in your terminal.
+
+## Step 5: Use Radius to display the state of your application
+
+Now that Radius has been enabled for your application, run this command once again:
```
Comment (jonvsm) [inline docs/content/tutorials/tutorial-add-radius/index.md]:
```suggestion
Now that Radius has been enabled for your application, run this command again:
```

File: docs/content/tutorials/tutorial-add-radius/index.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+```bash
+rad app connections -a demo
+```
+
+You should see now see some output containing information and status of your newly Radified applicaiton, similar to:
```
Comment (jonvsm) [inline docs/content/tutorials/tutorial-add-radius/index.md]:
```suggestion
You should see output containing information and status of your newly "radified" application, similar to:
```

**Review by jonvsm — APPROVED** [review]
Minor edits suggested for clarity

File: docs/content/tutorials/tutorial-add-radius/index.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
+---
+type: docs
+title: "Tutorial: Add Radius to an existing application"
+linkTitle: "Existing applications"
+description: "Learn how to incrementally add Radius to an existing application"
+weight: 600
+slug: "add-radius"
+categories: "Tutorial"
+tags : ["incremental","adoption"]
```
Comment (rynowak) [inline docs/content/tutorials/tutorial-add-radius/index.md]: I hope we don't use these as tags. Who would click on them?

Comment (willtsai) [reply docs/content/tutorials/tutorial-add-radius/index.md]: good point, I will change this.

Comment (willtsai) [inline docs/content/tutorials/tutorial-add-radius/index.md]:
```suggestion
tags : ["kubernetes","kubectl"]
```

File: docs/content/tutorials/tutorial-add-radius/index.md [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+- [Kubernetes cluster]({{< ref "guides/operations/kubernetes/overview#supported-kubernetes-clusters" >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [rad CLI]({{< ref howto-rad-cli >}})
+
+## Overview
+
+As a part of this tutorial you will deploy an existing containerized Guestbook application originally authored by the Kubernetes community for use in their own tutorials. You will then add Radius to the deployed application. To incrementally add Radius to an existing application, you will leverage the built-in Kubernetes integration functionality by adding an annotation to the application's Kubernetes deployment manifest. This approach is particularly useful for adding Radius capabilities to applications that are already deployed.
+
+The Guestbook application consists of a web front end along with primary and secondary Redis containers for storage, all deployed with Kubernetes. For more information about the application and access its source code, see the [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/stateless-application/guestbook/) and their [examples repo](https://github.com/kubernetes/examples/tree/master/guestbook).
+
```
Comment (rynowak) [inline docs/content/tutorials/tutorial-add-radius/index.md:32]: I would be good to have a diagram here if possible. We've consistently gotten feedback on tutorials that the diagram is needed.

Comment (rynowak) [reply docs/content/tutorials/tutorial-add-radius/index.md:32]: (non-blocking for this PR)

File: docs/content/tutorials/tutorial-add-radius/index.md [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+      annotations:
+         radapp.io/enabled: 'true'
+      ...
+   spec:
+   ...
+   ```
+
+   You should add the annotation to the following files:
+      - `deploy/frontend-deployment.yaml`
+      - `deploy/frontend-service.yaml`
```
Comment (rynowak) [inline docs/content/tutorials/tutorial-add-radius/index.md]: This doesn't do anything for the services.

Comment (willtsai) [reply docs/content/tutorials/tutorial-add-radius/index.md]: Good point, then it probably makes sense to exclude `Services` from the set of items to modify. I will make the change.

**Review by rynowak — APPROVED** [review]

**Pull request merged**